static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -386,6 +379,16 @@ require.def(
             getSeekableRange: function () {
                 return this._range;
             },
+            
+            isLiveRangeOutdated: function () {
+                var time = Math.floor(this._currentTime);
+                if (time % 8 === 0 && !this._updatingTime && this._lastWindowRanged !== time) {
+                    this._lastWindowRanged = time;
```

Comment:
"Hybrid" range update model proved to be necessary during our internal CATAL tests. Based on range seeking position is calculated so just manual updating time based on currentTime does not ensure that there will not be jumps out of range (range start) in aging test after long playback.

Current model actually ensures that there is no more than 1 range update every 8 seconds in terms of both manual and platform API updates.